ci-operator/config/openshift/hac-core/openshift-hac-core-main.yaml

```diff
@@ -13,9 +13,21 @@ resources:
       memory: 200Mi
 tests:
 - as: test
-  commands: ./test.sh
-  container:
-    from: src
+  steps:
+    test:
+    - as: test
+      commands: "export CODECOV_TOKEN=$(cat /tmp/secret/CORE_CODECOV_TOKEN) \n./test.sh\n"
+      credentials:
+      - mount_path: /tmp/secret
+        name: hac-core-deploy-tokens
+        namespace: test-credentials
+      from: src
+      resources:
+        limits:
+          memory: 4Gi
+        requests:
+          cpu: 100m
```

Comment:
@karelhala FYI during the investigation into the dynamic-plugin-sdk test issues (when we noticed `-w 1` fixes the issues with Jest in the pipeline), we noticed these values seemed low WRT `request` resources. We bumped the requested memory in the sdk: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/dynamic-plugin-sdk/openshift-dynamic-plugin-sdk-main.yaml#L30

For now this seems ok, but we may need to follow up on it.